db/R/dbfiles.R

```diff
@@ -147,18 +147,17 @@ dbfile.input.insert <- function(in.path, in.prefix, siteid, startdate, enddate,
 ##' @param con database connection object
 ##' @param hostname the name of the host where the file is stored, this will default to the name of the current machine
 ##' @param params database connection information
-##' @param exact.dates setting to include start and end date in input query
```

Comment:
OK, fixed it so that there are only the 2 files I _actually_ meant to include in this PR.